app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    first_file = command.split(" ")[1]
+    second_file = command.split(" ")[2]
```

Comment:
Use unpacking instead of indexing. Remember that space is a default separator for split and there is no need to explicitly indicate that
```suggestion
    command_name, first_file, second_file, *_ = command.split()
```